format/view-spec.md

```diff
@@ -243,7 +243,7 @@ Each change creates a new metadata JSON file.
 ```sql
 USE prod.other_db;
 CREATE OR REPLACE VIEW default.event_agg (
-    event_count,
+    event_count COMMENT 'Count of events',
```

Comment:
When compared this to line 175-177, 
column comment and view comment both were missing. But this change was not reflected back in the new metadata file.

- I updated the metadata file to remove properties. Since view comment is missing.
- Instead of adding a new schema id with missing doc, I added back the column comment to simplify the example.